docs/onboarding/onboarding-feature.md

```diff
@@ -0,0 +1,173 @@
+---
+id: onboarding-feature
+title: Onboarding feature
+slug: /onboarding
+---
+
+# Introduction
+The onboarding feature enables experimentation with the 'new user onboarding flow'. The onboarding flow is presented to the user on each new install and is made up of a series of full screen 'views', referred to as '**cards**'. The purpose of the onboarding flow is to enable the user to configure a small number of app enhancing settings. Each card provides context for each setting and the ability to enable/skip.
+
+The onboarding feature enables staff — most likely experiment owners, product owners, user research and marketing teams to customize each card's:
+
+- title copy
+- body copy
+- imagery
+- button copy
+- sequencing
+
+# About this document
+This document is a guide for staff who wish to configure the new user onboarding flow through the experimenter interface.
+
+It is also a living document:
+
+- the onboarding feature may be under active development
+- card types may be added
+- card attributes may be added 
+
+# Scene setting
+
+The onboarding feature is built on top of Nimbus, Mozilla's experimentation platform. Nimbus allows you to send bits of configuration to application features from Experimenter, the web-application staff use to launch and manage experiments and rollouts.
+
+Using Experimenter in the general case is documented elsewhere, so this document is specifically concerned with configuring the onboarding feature. [Example experiment](https://experimenter.services.mozilla.com/nimbus/lifestyles-images-onboarding-experiment/summary)
+
+
+# References
+For the most up-to-date configurations, the main code base will always be the best place to check.
+- [Nimbus manifest](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/nimbus.fml.yaml)
+- [onboarding feature manifest](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/onboarding.fml.yaml)
+
+The onboarding feature is a **first run experiment**. 
+- [for information on first run experiment](https://experimenter.info/mobile-first-run-experiments)
+- [for information on localization](https://experimenter.info/mobile-messaging/#localization-of-messages)
```

Comment:
`messaging` is a different feature: are the same localization concepts available in the onboarding feature?

Reply:
No, good spot, I'll remove this and update how localization will work for onboarding